base_report_to_printer/security/security.xml

```diff
@@ -1,116 +1,126 @@
 <?xml version="1.0" ?>
 <odoo noupdate="1">
```

Comment:
Remove noupdate="1"

Reply:
I agree, but this will not change on already installed databases.
I suggest to change the noupdate flag with a pre-migration script if we decide to remove noupdate here.

Reply:
There was a `<data>` tag after that has been removed now, so the final value was `noupdate="0"`, and thus no need of migration script:

![seleccion_017](https://user-images.githubusercontent.com/7165771/42020282-b6b37374-7ab7-11e8-9a5c-0ec246b15b8d.png)

Reply:
Oh, ok, I never encountered a `<data>` tag without noupdate in a `<odoo noupdate="1">` tag before. We learn a lot in this PR :)

Reply:
If we remove `noupdate=1` then might be chance to lost updated value when we upgrade module in same database.  :confused:

Reply:
No, as said, this has no effect, as there was after a `<data>` tag. Now that you are removing that tag, you have to remove the `noupdate` for letting the same situation.